src/components/scenes/PasswordRecoveryScene.tsx

```diff
@@ -1,47 +1,34 @@
-import { EdgeAccount, EdgeContext } from 'edge-core-js'
 import { PasswordRecoveryScreen } from 'edge-login-ui-rn'
 import * as React from 'react'
 
+import { useHandler } from '../../hooks/useHandler'
 import { config } from '../../theme/appConfig'
-import { connect } from '../../types/reactRedux'
+import { useSelector } from '../../types/reactRedux'
 import { EdgeSceneProps } from '../../types/routerTypes'
 import { logEvent } from '../../util/tracking'
 import { SceneWrapper } from '../common/SceneWrapper'
 
-interface OwnProps extends EdgeSceneProps<'passwordRecovery'> {}
+interface Props extends EdgeSceneProps<'passwordRecovery'> {}
 
-interface StateProps {
-  account: EdgeAccount
-  context: EdgeContext
-}
-type Props = StateProps & OwnProps
+export const ChangeRecoveryScene = (props: Props) => {
+  const { navigation } = props
+  const account = useSelector(state => state.core.account)
+  const context = useSelector(state => state.core.context)
 
-class ChangeRecoveryComponent extends React.Component<Props> {
-  render() {
-    const { context, account, navigation } = this.props
-    const handleComplete = () => navigation.goBack()
+  const handleComplete = useHandler(() => navigation.goBack())
 
-    return (
-      <SceneWrapper>
-        <PasswordRecoveryScreen
-          branding={{ appName: config.appName }}
-          account={account}
-          context={context}
-          onLogEvent={logEvent}
-          onComplete={handleComplete}
-          // @ts-expect-error
-          onCancel={handleComplete}
-          showHeader={false}
-        />
-      </SceneWrapper>
-    )
-  }
+  return (
+    <SceneWrapper>
+      <PasswordRecoveryScreen
+        branding={{ appName: config.appName }}
+        account={account}
+        context={context}
+        onLogEvent={logEvent}
+        onComplete={handleComplete}
+        // @ts-expect-error
+        onCancel={handleComplete}
```

Comment:
The reason we have `@ts-expect-error` is that there is no `onCancel` prop. So you could just delete this one.

Reply:
Not resolved yet.

Reply:
Accidentally squashed my fixup earlier and missed this in the re-rebase to split it up